src/World.h

```diff
@@ -416,6 +416,8 @@ class cWorld :
 	/** Calls the callback for each chunk in the coords specified (all cords are inclusive). Returns true if all chunks have been processed successfully */
 	virtual bool ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinChunkZ, int a_MaxChunkZ, cChunkDataCallback & a_Callback) override;
 
+    /** Calls the callback for each loaded chunk. Returns true if all chunks have been processed successfully */
+    virtual bool ForEachLoadedChunk(cChunkDataCallback & a_Callback);
```

Comment:
In fact is wrong everywhere. Please use tabs in front of each line, and spaces afterwards.